### PR TITLE
[WR118] Update Application Role

### PR DESCRIPTION
**Pull request by house9**
Implements `updateRealmApplicationRole`

Builds `dist` for both 'application role' and 'group'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -265,6 +265,58 @@ const roleId = '000000000000-0000-0000-0000-00000000'
 await accountClient.deleteRealmRole(realmName, roleId)
 ```
```
Comment (house9) [inline README.md:267]: Not sure if we should add additional documentation in this section explicitly calling out that 'Realm Application Role' CRUD uses the role name and not an id like 'Realm Role'? or just let the examples speak for themselves?

File: src/__tests__/realmApplicationRoles.test.ts [hunk 2]
```diff
@@ -75,7 +75,7 @@ describe('Realm Application Roles', () => {
       realmName,
       applicationId,
       {
-        name: 'Chef',
+        name: 'Head Chef',
```
Comment (house9) [inline src/__tests__/realmApplicationRoles.test.ts:78]: switched all tests to include space character which caused tests to fail until we added the encoding to the route

File: README.md (hunk 1 above)
Comment (pirtleshell) [reply README.md:267]: ah, lovely 😆

i'm happy to let the examples & function signatures speak for themselves.

**Review by pirtleshell — APPROVED** [review]
nice catch on encoding the role name 👍
TIL roles can have spaces ✨

**Review by efabens — APPROVED** [review]

**Review by galxy25 — APPROVED** [review]

**Pull request merged**